### PR TITLE
WT-12994 Fix parent validation to ignore cleared transaction ids

### PR DESCRIPTION
**Pull request by quchenhao**

### CODE REVIEW

File: src/support/timestamp.c [hunk 1]
```diff
@@ -413,7 +413,7 @@ __time_value_validate_parent(
           __wt_time_window_to_string(tw, time_string[0]),
           __wt_time_aggregate_to_string(parent, time_string[1]));
 
-    if (tw->start_txn > parent->newest_txn)
+    if (parent->newest_txn != WT_TXN_NONE && tw->start_txn > parent->newest_txn)
         WT_TIME_VALIDATE_RET(session,
           "value time window has a start transaction after its parent's newest transaction; "
```
Comment (etienneptl) [inline src/support/timestamp.c:418]: Why dont we see those messages in the reproducer?

Comment (quchenhao) [reply src/support/timestamp.c:418]: What do you mean? I saw those messages.

Comment (etienneptl) [reply src/support/timestamp.c:418]: My bad, I looked at the wrong file.

**Review by etienneptl — APPROVED** [review]

**Review by kommiharibabu — APPROVED** [review]
LGTM

**Pull request merged**